common/cross_os_debug/xdbg.cpp

```diff
@@ -471,14 +471,12 @@ bool check_stopped(const ThreadID& tid, SignalInfo* out) {
       {
         auto exc = debugEvent.u.Exception.ExceptionRecord.ExceptionCode;
         if (is_other) {
-          ContinueDebugEvent(debugEvent.dwProcessId, debugEvent.dwThreadId,
-                             DBG_EXCEPTION_NOT_HANDLED);
-          // if (exc == EXCEPTION_BREAKPOINT) {
-          //   out->kind = SignalInfo::BREAK;
-          // } else {
-          //   // ignore exceptions outside goal thread
-          //   ignore_debug_event();
-          // }
+          if (exc == EXCEPTION_BREAKPOINT) {
+            out->kind = SignalInfo::BREAK;
+          } else {
+            // ignore exceptions outside goal thread
+            ignore_debug_event();
```

Comment:
I think the original reason to change this was because there were uncaught exceptions being thrown in the graphics thread, and this somehow caused them to be ignored, but make the game run extremely slowly.